src/styles/blocks/product-card.scss

```
@@ -0,0 +1,94 @@
+$color-light-blue: rgba(0, 172, 220, 1);
+$title-line-height: 18px;
+
+.product-card {
+  margin-bottom: 46px;
+  margin-right: 48px;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  &:nth-child(n + 5) {
+    margin-bottom: 0;
+  }
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__image {
+    width: 100%;
+  }
+
+  &__title {
+    font-size: 12px;
+    margin-top: 40px;
+    line-height: 18px;
+    font-weight: 500;
+    color: rgba(6, 11, 53, 1);
+  }
+
+  &__subtitle {
+    color: rgba(97, 96, 112, 1);
+    font-weight: 400;
+    font-size: 10px;
+    margin-top: 4px;
+  }
+
+  &__rates {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+  }
+
+  &__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    margin-top: 4px;
+    color: rgb(6, 11, 53);
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-content: center;
+    margin-top: 24px;
+
+    &-title {
+      font-size: 12px;
+      line-height: #{$title-line-height};
+      font-weight: 400;
+      color: #616070;
```

don't forget to create variables for all colors
```suggestion
      color: $c-color;
```